LocationManager/INTULocationManager/INTUHeadingRequest.m

```diff
@@ -0,0 +1,78 @@
+//
+//  INTUHeadingRequest.m
+//
+//  Copyright (c) 2014-2015 Intuit Inc.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining
+//  a copy of this software and associated documentation files (the
+//  "Software"), to deal in the Software without restriction, including
+//  without limitation the rights to use, copy, modify, merge, publish,
+//  distribute, sublicense, and/or sell copies of the Software, and to
+//  permit persons to whom the Software is furnished to do so, subject to
+//  the following conditions:
+//
+//  The above copyright notice and this permission notice shall be
+//  included in all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+//  EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+//  MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+//  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+//  LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+//  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+//  WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+#import "INTUHeadingRequest.h"
+
+@implementation INTUHeadingRequest
+
+static INTUHeadingRequestID _nextRequestID = 0;
```

Comment:
This just occurred to me: we should make sure that INTULocationRequest and INTUHeadingRequest are returning unique request IDs (integers), even between the two. If the IDs overlap, it's likely to cause bugs due to (unintentional) misuse by clients. So, let's factor out the underlying static variable and its associated function here (perhaps into a very small INTURequestIDGenerator file or something) and have it shared between both classes.